baselines/reference/singleVar.ts.errors.baseline

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
these baselines are the same as singleVar.*.baseline except they have an additional .ts in the filename